ansible-scylla-node/tasks/Debian.yml

```diff
@@ -61,6 +61,13 @@
     with_items: "{{ scylla_deb_repos }}"
     when: install_type == 'online' and item.split(".")[-1] == "list"
 
+  - name: Update gnupg2 dependency if necessary
```

Comment:
Why is this needed?

Reply:
Without this `gnupg2` will never be updated by the role, since now the previous task installing `gnupg2` will only be executed if it's not already installed.

Reply:
Sure and this is how it was so far. Why is this a problem?

Reply:
Fixed